nixos/modules/module-list.nix

```diff
@@ -629,4 +629,5 @@
   ./virtualisation/vmware-guest.nix
   ./virtualisation/xen-dom0.nix
   ./virtualisation/xe-guest-utilities.nix
+  ./virtualisation/keystone.nix
```

Comment:
IMHO, I think keystone is not about virtualization at all.
My preference would be to either:
- move separate services to categories that match what they do
- create sub-category in virtualisation for open stack and keep everything there, i.e.
```
virtualisation.openstack = {
  keystone.enable=true;
};
```